ci_config.json

```diff
@@ -44,6 +44,11 @@
       "windows": false
     }
   },
+  "flac" : {
+    "build_options": [
+      "c_std=gnu99"
```

Comment:
Anyone have any idea why this is not being passed?

Reply:
Seems like it is:
```
FAILED: subprojects/flac-1.3.4/src/libFLAC/libFLAC.8.dylib.p/metadata_object.c.o 
cc -Isubprojects/flac-1.3.4/src/libFLAC/libFLAC.8.dylib.p -Isubprojects/flac-1.3.4/src/libFLAC -I../subprojects/flac-1.3.4/src/libFLAC -I../subprojects/flac-1.3.4/include -Isubprojects/flac-1.3.4 -I../subprojects/flac-1.3.4 -I../subprojects/flac-1.3.4/src/libFLAC/include -I/usr/local/Cellar/libogg/1.3.5/include -fcolor-diagnostics -Wall -Winvalid-pch -std=gnu99 -O0 -g -DHAVE_CONFIG_H -D_GNU_SOURCE -DFLAC_API_EXPORTS -DFLAC__USE_VISIBILITY_ATTR -MD -MQ subprojects/flac-1.3.4/src/libFLAC/libFLAC.8.dylib.p/metadata_object.c.o -MF subprojects/flac-1.3.4/src/libFLAC/libFLAC.8.dylib.p/metadata_object.c.o.d -o subprojects/flac-1.3.4/src/libFLAC/libFLAC.8.dylib.p/metadata_object.c.o -c ../subprojects/flac-1.3.4/src/libFLAC/metadata_object.c
../subprojects/flac-1.3.4/src/libFLAC/metadata_object.c:117:15: error: implicitly declaring library function 'strdup' with type 'char *(const char *)' [-Werror,-Wimplicit-function-declaration]
        char *copy = strdup(from);
                     ^
../subprojects/flac-1.3.4/src/libFLAC/metadata_object.c:117:15: note: include the header <string.h> or explicitly provide a declaration for 'strdup'
../subprojects/flac-1.3.4/src/libFLAC/metadata_object.c:1386:76: error: implicitly declaring library function 'strncasecmp' with type 'int (const char *, const char *, unsigned long)' [-Werror,-Wimplicit-function-declaration]
                return (eq != NULL && (uint32_t)(eq-entry.entry) == field_name_length && FLAC__STRNCASECMP(field_name, (const char *)entry.entry, field_name_length) == 0);
                                                                                         ^
../subprojects/flac-1.3.4/include/share/compat.h:98:27: note: expanded from macro 'FLAC__STRNCASECMP'
#define FLAC__STRNCASECMP strncasecmp
                          ^
../subprojects/flac-1.3.4/src/libFLAC/metadata_object.c:1386:76: note: include the header <strings.h> or explicitly provide a declaration for 'strncasecmp'
../subprojects/flac-1.3.4/include/share/compat.h:98:27: note: expanded from macro 'FLAC__STRNCASECMP'
#define FLAC__STRNCASECMP strncasecmp
                          ^
2 errors generated.
```


-Wall -Winvalid-pch -std=gnu99 -O0 -g